src/app/feature/template/template.page.ts

```diff
@@ -45,7 +46,11 @@ export class TemplatePage implements OnInit, OnDestroy {
   private subscribeToAppConfigChanges() {
     this.appConfigChanges$ = this.appConfigService.changesWithInitialValue$.subscribe(
       (changes: IAppConfig) => {
-        if (changes.APP_HEADER_DEFAULTS?.collapse !== undefined) {
+        // TODO: current header collapse implementation does not work on ios, so do not enable on this platform
+        if (
+          changes.APP_HEADER_DEFAULTS?.collapse !== undefined &&
```

Comment:
The idea was that the `collapse` property can, at least in theory, change at runtime (via a skin change). So if it's value changed from `true` -> `false`, we'd want to update `shouldEmitScrollEvents` accordingly

Reply:
In that case I'd say just make the default value false so we only have to consider changes from `true` <->`false` instead of a fun 
```mermaid
graph TD;
    undefined-->TRUE;
    undefined-->FALSE;
    TRUE-->FALSE;
    FALSE-->TRUE;
```

(also totally forgot you can now embed mermaidjs diagrams in github flavoured markdown... PR reviews about to get a whole lot spicier🌶️ )

Reply:
Ooh very nice! Thanks, I've updated this PR, the if statement did have a whiff about it...